packages/namekit-react/src/components/EthereumCircleIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { type SVGProps } from "react";
+
+interface LogoProps extends SVGProps<SVGSVGElement> {
+  isSmallIcon?: boolean;
```

Comment:
Hmm.. my first impression is this prop feels like code smell. Can you please investigate the purpose of this prop in more detail and suggest a good solution for us? Thanks